server/monitor/pom.xml

```diff
@@ -137,7 +142,8 @@
           <artifactId>apache-rat-plugin</artifactId>
           <configuration>
             <excludes>
-              <exclude>src/main/resources/resources/js/flot/**/*.js</exclude>
+              <exclude>src/main/resources/resources/external/*</exclude>
```

Comment:
Would one line of the following be better?
```xml
<exclude>src/main/resources/resources/external/**/*</exclude>
```
This will match all files in the `external` directory, including any subdirectories.

Reply:
For sure. Forgot about this change, thanks.